CHANGELOG.md

```diff
@@ -1,6 +1,247 @@
 Changelog
 =========
 
+1.8.0
+-----
+
+### IMPORTANT NOTES
+
+#### Last release before a PHP minimum version bump
+
+  Release 1.8.0 will be the last release with compatibility for PHP 5.2 - 5.5. With the next release (v2.0.0), the minimum PHP version will be bumped to 5.6+, matching the current PHP minimum version requirement of WordPress Core.
+
+#### Last release supporting PEAR distribution
+
+  Release 1.8.0 will be the last release to be distributed via PEAR. From release 2.0.0 onwards, consumers of this library will have to switch to Composer to receive updates.
+
+### Overview of changes
+
+- **[SECURITY FIX] Disable deserialization in `FilteredIterator`**
+
+  A `Deserialization of Untrusted Data` weakness was found in the `FilteredIterator` class.
+  
+  This security vulnerability was first detected in the files bundled with WordPress. The security fix from https://core.trac.wordpress.org/changeset/49373 has now been ported back into the library.
+  
+  GitHub security advisory: [link to GHSA]
+
+  Related WordPress CVE: https://cve.mitre.org/cgi-bin/cvename.cgi?name=2020-28032
+
+  (props [@dd32][gh-dd32], [@desrosj][gh-desrosj], [@jrfnl][gh-jrfnl], [@peterwilsoncc][gh-peterwilsoncc], [@SergeyBiryukov][gh-SergeyBiryukov], [@whyisjake][gh-whyisjake], [@xknown][gh-xknown], [#421][gh-421], [#422][gh-422], [#457][gh-457])
+
+
+- **Repository moved to `WordPress\Requests`**
+
+  The `Requests` library has been officially adopted by the WordPress project
```

Comment:
Are you referring to the GitHub organisation here? That would just be a move, not an adoption. What I was referring to was the open source WordPress project, which - if I understand correctly - has adopted the library to help maintain it. That project has no legal entity directly, so organisation would be misleading.

Reply:
Isn't the WordPress Foundation the organisation behind the open source WordPress project ? That's more what I meant.

I.e. this project has now become one of the projects supported/maintained by that organisation and its open source contributors and the repo has been moved to the matching GH organisation.

I think the word "project" can easily be misinterpreted as the WordPress open source project, which would give the impression that WordPress has swallowed this project instead of it living as a stand-alone project - albeit used by the WordPress project - within the WordPress organisation.

Reply:
Here's how I understand it:

The WordPress Foundation is maintaining the trademark for the `WordPress` name and (through WordPress Community Support) manages the financials around sponsorships and events.

As you can see on the foundation's ["About" page](https://wordpressfoundation.org/), it supports the `WordPress open source project`, which is the "entity" that includes multiple pieces of software, like WordPress Core, the WordPress meta environment, the Gutenberg plugin, the WordPress Coding Standards, etc...

The `Requests` library has joined the WordPress open source project, but it is unrelated to the WordPress Foundation.

AFAIK, there was never a legal entity to cover the actual code that is maintained, it is just an open source GPL software project.